go.mod

```diff
@@ -144,4 +145,6 @@ require (
 	sigs.k8s.io/yaml v1.2.0 // indirect
 )
 
-go 1.17
+replace github.com/lightningnetwork/lnd => github.com/guggero/lnd v0.11.0-beta.rc4.0.20230111160525-4e3decd26baf
+
+go 1.18
```

Comment:
I think we will have to bump it to go `v1.19` after tagging  LND `v0.16.0-beta`

Reply:
We didn't bump this on the `lnd` either yet, so LGTM :tada: